### ISSUE TITLE
BackstageTabItem Button Icon not shown

### ISSUE THREAD

**Issue by WebSurfer1234** [fluentribbon/Fluent.Ribbon#679]
Hi,
I'm trying to append an icon to my BackstageTabItem Button, but the item is not shown.
I have tried to use icons in *.ico and *.png format (both 16x16px)

![image](https://user-images.githubusercontent.com/22983942/50148907-04403480-02ba-11e9-80ac-7d850c48228a.png)

This is my code:
```
<Fluent:BackstageTabItem Header="Options">
    <Fluent:Button Header="Testbutton" Foreground="black">
        <Fluent:Button.LargeIcon>
            <Image Source="Images/icon.png"></Image>
        </Fluent:Button.LargeIcon>
        <Fluent:Button.Icon>
            <Image Source="Images/icon.png"></Image>
        </Fluent:Button.Icon>
    </Fluent:Button>
</Fluent:BackstageTabItem>
```
---
### Environment

- Fluent.Ribbon __v6.1.0.326__
- Windows __10__
- .NET Framework __4.6.1__

Is that an issue, or did i do something wrong?

Best Regards

**Comment by batzen** [comment]
It was an issue and is fixed in version 7.0 which is available as a prerelease on nuget. The final release is planned for late december/early January.

**Comment by WebSurfer1234** [comment]
Ah ok got it thank you very much. :-)
